web-dapp/server-lib/logger.js

```diff
@@ -1,10 +1,30 @@
+'use strict';
+const winston = require('winston');
+
+const transports = [];
+
+if (process.env.NODE_ENV !== 'test') {
+    // Console transport
+    const consoleTransport = new winston.transports.Console({
+        level: process.env.LOG_LEVEL || 'debug',
+        colorize: true,
```

Comment:
I suggest we turn off coloring, because it adds special symbols to output that are interpreted by terminal as color codes, however if terminal does not support them, things become less readable and less grep-able

Reply:
👍